js/widgets/fixedToolbar.js

```diff
@@ -80,17 +81,23 @@
 				this._makeFixed();
 			}
 			if ( this.options.position === "fixed" && !this.options.supportBlacklist() ) {
-				var $page = ( !!this.page )? this.page: ( $(".ui-page-active").length > 0 )? $(".ui-page-active"): $(".ui-page").eq(0);
+				var activePage = $( ".ui-page-active" );
```

Comment:
add a Todo: here once we land `pagecontainers` to use set the current pagecontainer in create as `this.element.closest( pagecontainers );` then to get the active page will be `this.pagecontainer.getActive()`